kratos.gid/scripts/Writing/Writing.tcl

```diff
@@ -577,6 +594,22 @@ proc write::mdpaIndent { {b 4} } {
     string repeat [string repeat " " $b] $current_mdpa_indent_level
 }
 
+# Sets the precission for the diffetent entities written in the mdpa
+proc write::InitWriteFormats { } {
+    variable formats_dict
+    if {$::Kratos::kratos_private(mdpa_format) == 1} {
+        dict set formats_dict ID "%5d"
+        dict set formats_dict CONNECTIVITY "%10d"
+        dict set formats_dict MAT_ID "%10d"
+        dict set formats_dict COORDINATE "%14.10f"
+    } else {
+        dict set formats_dict ID "%d"
+        dict set formats_dict CONNECTIVITY "%d"
+        dict set formats_dict MAT_ID "%d"
+        dict set formats_dict COORDINATE "%.10f"

```

Comment:
@philbucher  @loumalouomega 
Here is where the nodal coordinates precission is set. Just in case someone needs more than 10 decimals

Reply:
Could you add a comment in the code explaining this?

Reply:
Better @rubenzorrilla ?

https://github.com/KratosMultiphysics/GiDInterface/pull/667/commits/2f2ba3c8f03fa46ce6449eeca75d6c752d81b087